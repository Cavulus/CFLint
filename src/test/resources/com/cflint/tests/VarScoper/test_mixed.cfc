
<cfcomponent>           
  <cffunction name="foo">  
  <cfscript>     
    x = 123;
  </cfscript>
  </cffunction>  
</cfcomponent>