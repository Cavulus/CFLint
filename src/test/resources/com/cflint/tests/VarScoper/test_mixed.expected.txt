{
  "version" : "",
  "timestamp" : 1527541303,
  "issues" : [ {
    "severity" : "ERROR",
    "id" : "MISSING_VAR",
    "message" : "MISSING_VAR",
    "category" : "CFLINT",
    "abbrev" : "MV",
    "locations" : [ {
      "file" : "src\\test\\resources\\com\\cflint\\tests\\VarScoper\\test_mixed.cfc",
      "fileName" : "test_mixed.cfc",
      "function" : "foo",
      "offset" : 76,
      "column" : 5,
      "line" : 5,
      "message" : "Variable x is not declared with a var statement.",
      "variable" : "x",
      "expression" : "x = 123"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "MISSING_VAR",
      "count" : 1
    } ],
    "countBySeverity" : [ {
      "severity" : "ERROR",
      "count" : 1
    } ]
  }
}